HuongDoiTuong/BaiTap6.playground/Contents.swift

```diff
@@ -0,0 +1,62 @@
+
+//Thư viện cần quản lý việc mượn, trả sách. Sinh viên đăng ký tham gia mượn sách thông qua thẻ mà thư viện cung cấp.
+//Với mỗi thẻ sẽ lưu các thông tin sau: Mã phiếu mượn, ngày mượn(kiểu Int), hạn trả, số hiệu sách, và các thông tin cá nhân của sinh viên mượn sách. Các thông tin của sinh viên mượn sách bao gồm: Họ tên, tuổi, lớp.
+//Yêu cầu 1: Xây dựng lớp SinhVien để quản lý thông tin của mỗi sinh viên.
+//Yêu cầu 2: Xây dựng lớp TheMuon để quản lý việc mượn trả sách của các sinh viên.
+//Yêu cầu 3: Xây dựng lớp QuanLy có các phương thức: Thêm, xoá theo mã phiếu mượn và hiển thị thông tin các thẻ mượn.
+
+class SinhVien {
+    var hoTen: String
+    var tuoi: Int
+    var lop: String
+    var maPhieuMuon: Int
```

Comment:
bao gồm: Họ tên, tuổi, lớp.
pls remove maPhieuMuon, it is not a property of class SinhVien